.github/workflows/python-publish.yml

```
@@ -47,14 +51,14 @@ jobs:
      needs: test
      steps:
        - uses: actions/checkout@v4
-       - name: Set up Python
+       - name: Set up Python ${{ matrix.python-version }}
          uses: actions/setup-python@v5
          with:
            python-version: ${{ matrix.python-version }}
        - name: Install dependencies
          run: |
            python -m pip install --upgrade pip
-           pip install -r dev_requirements/requirements-test_packaging.txt
+           pip install .[test_packaging]
```

```suggestion
           pip install './.[test_packaging]'
```
so hat es bei mir auch lokal geklappt

---

was genau die suggestion macht verstehe ich nicht, aber jetzt funktiniert das auch schon
https://github.com/Hochfrequenz/kohlrahbi/actions/runs/11032352423

---

dann ziehe ich meine suggestions hiermit zurück :)